src/renderer/components/+namespaces/namespace-select-filter.tsx

```diff
@@ -74,26 +76,42 @@ export class NamespaceSelectFilter extends React.Component<SelectProps> {
   }
 
   onChange([{ value: namespace }]: SelectOption[]) {
-    if (namespace) {
+    if (NamespaceSelectFilter.isMulti && namespace) {
       namespaceStore.toggleContext(namespace);
+    } else if (!NamespaceSelectFilter.isMulti && namespace) {
+      namespaceStore.toggleSingle(namespace);
     } else {
       namespaceStore.toggleAll(false); // "All namespaces" clicked
```

Comment:
I don't think we should have "All Namespaces" anymore.

Reply:
@jakolehm Is this fine?

Reply:
I think we still need some sort of "Select All/Clear All" option. Equivalent to Ctrl-A. To select just my 200 test namespaces I can "Select All" then deselect my 6 normal namespaces. 

Or maybe we need to be able to select ranges with modified clicks? Windows and Ubuntu commonly use shift-click to select the range between last clicked and current clicked. Ctrl-click adds individual items to the multi-selection. And Ctrl-Shift-click adds a range to the multi-selection.

I also think when all namespaces are selected they should all have check marks (instead of none having check marks, as it is now). Or a check mark beside "All namespaces"

Reply:
Well we could do that with shift-clicking the last remaining.

And then if you want to remove them you could shift-click the ones you want to remove. 

That way "shift-clicking" means toggle if this namespace is in the output set.

Reply:
We definitely need a way to select/deselect "All Namespaces".

Reply:
Thank you guys for your comments. I will  leave "All namespaces" as it is and will add closing the namespace select if user selects just one namespace, do I understand you correctly?

Reply:
@jakolehm Should `"All Namespaces"` be available if `global watch` is `false`?

Reply:
> @jakolehm Should `"All Namespaces"` be available if `global watch` is `false`?

Maybe not but hiding "All Namespaces" should be done in a separate PR.